jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Fields.java

```diff
@@ -246,13 +248,30 @@ public void add(String name, String... values)
             fields.compute(name, (k, f) ->
             {
                 if (f == null)
-                    return new Field(name, List.of(values));
+                    return new Field(name, toList(values));
                 else
-                    return new Field(f.getName(), f.getValues(), List.of(values));
+                    return new Field(f.getName(), f.getValues(), toList(values));
             });
         }
     }
 
+    static List<String> toList(String... strings)
+    {
+        try
+        {
+            return List.of(strings);
+        }
+        catch (NullPointerException e)
```

Comment:
Don't really like exceptions as control flow... Not really on the hot path, I'd go for `ArrayList` and iteration over the array.

Reply:
Done.